apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/Oppgavelista.tsx

```diff
@@ -182,49 +158,41 @@ export const ToggleMinOppgaveliste = () => {
     }
   }, [hentRevurderingsaarsakerStatus])
 
+  useEffect(() => {
+    leggFilterILocalStorage({ ...oppgavelistaFilter, sakEllerFnrFilter: '' })
+  }, [oppgavelistaFilter])
+
+  useEffect(() => {
+    leggValgILocalstorage(oppgavelisteValg)
+  }, [oppgavelisteValg])
```

Comment:
Jeg ville muligens gått for searchParams i stedet, men kan teste ut localstorage og se hva de synes. Er uansett fordeler og ulemper med begge. 🤷‍♂️